Improve grammar, spelling, and punctuation in Getting started section (#9394)
wagtail/wagtail: docs/getting_started/tutorial.md

## A basic blog

We are now ready to create a blog. To do so, run

> `python manage.py startapp blog` to create a new app on your Wagtail site.

Reviewer: "on your Wagtail site" would be more natural if we were talking about an area of a website that someone can visit, but in this case it's referring to the codebase. So maybe "project" is more accurate than "site"...

```suggestion
`python manage.py startapp blog` to create a new app in your Wagtail project.
```
